inc/config.php

```diff
@@ -211,16 +211,17 @@
                         $newer = true;
                      }
                   }
-               } else if (strlen($CFG_GLPI['dbversion']) > 40) {
+               } else if (!isset($CFG_GLPI['dbversion']) || strlen($CFG_GLPI['dbversion']) > 40) {
```

Comment:
Only the following test should be added. The case where `$CFG_GLPI['dbversion']` is not set is already handled on line 222.
```suggestion
               } else if (isset($CFG_GLPI['dbversion']) && strlen($CFG_GLPI['dbversion']) > 40) {
```

Reply:
in 222 it's not the same test

214 => test of lenght of dbversion
222 => test of value of dbversion